first_ruby/config/locales/en.yml

```diff
@@ -0,0 +1,33 @@
+# Files in the config/locales directory are used for internationalization
```

Comment:
1. Whats this file?
2. How do you use this file?

Reply:
1. To define the key and translation for the locale, it can also display the localize date and the defined format. 
2. In this case, the configured locale is `en` and the setting is on key `hello`
```
en:
    hello: "Hello world!"
```
To use this, in the `view` use the format `<%= t('hello') %>` and this will display `Hello world!` at the page when display.